helm/jsonapi/templates/stargate_jsonapi_deployment.yaml

```diff
@@ -1,62 +1,195 @@
-          failureThreshold: {{ .Values.probe.liveness.failureThreshold }}
-          initialDelaySeconds: {{ .Values.probe.liveness.initialDelaySeconds }}
-        resources:
-          requests:
-            cpu: {{ .Values.cpu | default 1000 }}m
-            memory: {{ .Values.memory | default 1024 }}Mi
-        env:
-        - name: QUARKUS_GRPC_CLIENTS_BRIDGE_HOST
-          value: "{{ .Values.sgGrpcBridge.serviceName }}.{{ .Values.sgGrpcBridge.namespace }}"
-        - name: IO_STARGATE_SGV2_API_COMMON_PROPERTIES_DATASTORE_CONFIGURATION_DATASTOREPROPERTIESCONFIGURATION_FETCHSUPPORTEDFEATURES_RETRY_MAXRETRIES
-          value: "50"
-        - name: IO_STARGATE_SGV2_API_COMMON_PROPERTIES_DATASTORE_CONFIGURATION_DATASTOREPROPERTIESCONFIGURATION_FETCHSUPPORTEDFEATURES_RETRY_MAXDURATION
-          value: "300"
-        - name: IO_STARGATE_SGV2_API_COMMON_PROPERTIES_DATASTORE_CONFIGURATION_DATASTOREPROPERTIESCONFIGURATION_FETCHSUPPORTEDFEATURES_RETRY_DELAY
-          value: "10"
+        - env:
+            - name: QUARKUS_HTTP_HOST
+              value: "0.0.0.0"
+            - name: QUARKUS_HTTP_PORT
+              value: "{{ .Values.api.port }}"
+            - name: QUARKUS_SHUTDOWN_TIMEOUT
+              value: "PT30S"
+            - name: QUARKUS_SMALLRYE_OPENAPI_SERVERS
+              value: /api/json
+            - name: QUARKUS_HTTP_ACCESS_LOG_ENABLED
+              value: "true"
+            - name: QUARKUS_HTTP_ACCESS_LOG_PATTERN
+              value: "{{ .Values.accessLogPattern }}"
+            - name: QUARKUS_LOG_MIN_LEVEL
+              value: "TRACE"
+            - name: QUARKUS_LOG_CONSOLE_JSON_EXCEPTION_OUTPUT_TYPE
+              value: formatted
+            {{- range $key, $value := .Values.logParams }}
+            {{- if hasPrefix "QUARKUS_LOG_" $key }}
+            - name: {{ $key }}
+              value: {{ $value }}
+            {{- end }}
+            {{- end }}
+            - name: STARGATE_DATA_STORE_IGNORE_BRIDGE
+              value: "true"
+            - name: STARGATE_DATA_STORE_SAI_ENABLED
+              value: "true"
+            - name: STARGATE_METRICS_TENANT_REQUEST_COUNTER_USER_AGENT_TAG_ENABLED
+              value: "true"
+            - name: STARGATE_MULTI_TENANCY_ENABLED
+              value: "true"
+            - name: STARGATE_MULTI_TENANCY_TENANT_RESOLVER_TYPE
+              value: "subdomain"
+            - name: STARGATE_MULTI_TENANCY_TENANT_RESOLVER_SUBDOMAIN_MAX_CHARS
+              value: "36"
+            - name: STARGATE_MULTI_TENANCY_TENANT_RESOLVER_SUBDOMAIN_REGEX
+              value: "[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}"
+            - name: JAVA_MAX_MEM_RATIO
+              value: "75"
+            - name: JAVA_INITIAL_MEM_RATIO
+              value: "50"
+            - name: GC_CONTAINER_OPTIONS
+              value: "-XX:+UseG1GC"
+            - name : STARGATE_JSONAPI_OPERATIONS_DATABASE_CONFIG_TYPE
+              value: astra
```

Comment:
really?

Reply:
did we test this helm chart?

Reply:
No, I referenced from what we have in serverless-platform-charts repo.
Let me refine it.